nixos/modules/module-list.nix

```
@@ -1184,6 +1184,7 @@
   ./services/system/saslauthd.nix
   ./services/system/self-deploy.nix
   ./services/system/systembus-notify.nix
+  ./services/system/systemd-lock-handler
```

```suggestion
  ./services/system/systemd-lock-handler.nix
```
normally we place the files like this